pvsc-dev-ext.py

```diff
@@ -132,15 +132,20 @@ def parse_args(args=sys.argv[1:]):
     subparsers = parser.add_subparsers(dest="cmd")
     setup_parser = subparsers.add_parser("setup")
     setup_parser.add_argument("install_type", choices=[install_type.name for install_type in VSCode])
+    setup_parser.add_argument('--repo', dest='repo', default=REPO_URL)
+    setup_parser.add_argument('--branch', dest='branch', default='master')
     update_parser = subparsers.add_parser("update")
     return parser.parse_args(args)
 
 
 if __name__ == "__main__":
     args = parse_args()
-    if args.cmd == "setup":
-        setup(VSCode[args.install_type])
-    elif args.cmd == "update":
-        update()
-    else:
-        raise RuntimeError(f"unrecognized sub-command: {args.cmd!r}")
+    try:
+        if args.cmd == "setup":
+            setup(VSCode[args.install_type], args.repo, args.branch)
+        elif args.cmd == "update":
+            update()
+        else:
+            raise RuntimeError(f"Unrecognized sub-command: {args.cmd!r}")
+    except subprocess.CalledProcessError as cpe:
+        print(f"Failed to run command {cpe.cmd} : {cpe.stderr}")
```

Comment:
```suggestion
        print(f"Failed to run command {exc.cmd} : {exc.stderr}")
```

I thought `CalledProcessError` included the output in its message, but I guess not. :/